.github/workflows/gatk-tests.yml

```diff
@@ -16,19 +19,29 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
```

Comment:
Instead of a million checks for if(secrets==yes) I almost want to pull this whole thing out as a composite action, and then just run a separate job for tests that need secrets.  That's probably not worth it though is it?

Reply:
maybe, I say punt on it though. We might want a cleanup round after we merge, but lets not hold it up.